tests/unit-tests/admin/reports/class-wc-tests-report-sales-by-date.php

```diff
@@ -96,7 +96,7 @@ public function test_get_report_data() {
 			$data->coupons[0]->order_item_name,
 			'There should be a single coupon applied.'
 		);
-		$this->assertEquals( $data->coupons[0]->discount_amount, $data->total_coupons, 'Total discount amount.' );
+		$this->assertEquals( wc_format_decimal( $data->coupons[0]->discount_amount, '' ), $data->total_coupons, 'Total discount amount.' );
```

Comment:
PHPUnit 7.5 is more strict, before strings like `"99"` are equal to `"99.00"`, but now we need to make strict as well.